users/views.py

```diff
@@ -1,3 +1,58 @@
+import json
+import re
+import jwt
+import bcrypt
+
 from django.shortcuts import render
+from django.views import View
+from django.http import JsonResponse
+from django.conf import settings
+from .utils import login_decorator
+from .models import User
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            email_check = re.compile('^[a-zA-Z0-9-_]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+$')
+            if not re.match(email_check, data['email']):
+                return JsonResponse({'message' : 'EMAIL_ERROR'}, status=400)
+
+            if len(data['password']) < 6:
+                return JsonREsponse({'message' : 'PASSWORD_ERROR'}, status=400)
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message' : 'EMAIL_ERROR'}, status=400)
+
+            password         = data['password'].encode('utf-8')
+            hashed_password  = bcrypt.hashpw(password, bcrypt.gensalt())
+            decode_password  = hashed_password.decode('utf-8')
+
+            User.objects.create(
+                name      = data['name'],
+                email     = data['email'],
+                password  = decode_password)
+            return JsonResponse({'message' : 'SUCCESS'}, status=200)
+
+        except Exception:
+            return JsonResponse({'message' : 'ERROR'}, status=400)
+
+class LoginView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if 'email' not in data:
```

Comment:
```suggestion
            if 'email' not in data or 'password' not in data:
```
에러코드에 맞게 둘다 검증하시면 좋을 것 같네요!